mobile/components/Lists/NotificationListItem.js

```diff
@@ -0,0 +1,290 @@
+// @flow
+import * as React from 'react';
+import { ListItemWithIcon } from './ListItemWithIcon';
+import {
+  TextColumnContainer,
+  TextRowContainer,
+  TimestampTextContainer,
+  Timestamp,
+  Subtitle,
+  Link,
+  FacepileContainer,
+} from './style';
+import type { ParsedNotificationType } from '../../views/notifications/parseNotification';
+import type { Navigation } from '../../utils/types';
+import { timeDifferenceShort } from '../../../shared/time-difference';
+import Facepile from '../Facepile';
+import Icon from '../Icon';
+
+type Props = {
+  notification: ParsedNotificationType,
+  children?: React.ChildrenArray<any>,
```

Comment:
I think you can make this `children?: React.Node` and it'll work without having to type it to `any`